src/ngMessages/messages.js

```diff
@@ -24,10 +24,16 @@ var jqLite = angular.element;
  * `ngMessage` and `ngMessageExp` directives.
  *
  * # Usage
- * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute.
- * Since the {@link ngModel ngModel} directive exposes an `$error` object, this error object can be
- * used with `ngMessages` to display control error messages in an easier way than with just regular angular
- * template directives.
+ * The `ngMessages` directive listens on a key/value collection which is set on the ngMessages attribute of a container element.
```

Comment:
This reads like the ngMessages attribute is on a container element that is not the same as the ngMessages element